tests/everflow/everflow_test_utilities.py

```diff
@@ -406,17 +501,19 @@ def setup_mirror_session(self, duthosts, rand_one_dut_hostname, config_method):
         Yields:
             dict: Information about the mirror session configuration.
         """
-        duthost = duthosts[rand_one_dut_hostname]
-        session_info = BaseEverflowTest.mirror_session_info("test_session_1", duthost.facts["asic_type"])
+        duthost_set = BaseEverflowTest.get_duthost_set(setup_info)
 
-        BaseEverflowTest.apply_mirror_config(duthost, session_info, config_method)
+        for duthost in duthost_set:
+            session_info = BaseEverflowTest.mirror_session_info("test_session_1", duthost.facts["asic_type"])
```

Comment:
Please put `session_info = BaseEverflowTest.mirror_session_info("test_session_1", duthost.facts["asic_type"])` outside the for loop.

Reply:
added check to invoke only one time